doc/src/modules/concrete.rst

```diff
@@ -15,7 +15,7 @@ method which is available in Basic class. Here is simple example involving a
 polynomial:
 
     >>> from sympy import *
-    >>> n, k = symbols('n,k')
+    >>> n, k = symbols('n,k', integer=True, nonnegative=True)
```

Comment:
Is this still necessary? If not then maybe it and the gosper changes could be undone.

Reply:
I just checked and `integer=True` can be removed but removing `nonnegative=True` seems to make `is_hypergeometric` return false. 
I'll add that to the TODO ? or is it expected behaviour @smichr ?

Reply:
What about having only `nonzero=True`?

Reply:
```python3
>>> n, k = symbols('n,k',nonnegative=True)
>>> binomial(n, k).is_hypergeometric(n)
True
>>> n, k = symbols('n,k',nonzero=True)
>>> binomial(n, k).is_hypergeometric(n)
False
>>> n, k = symbols('n,k',positive=True)
>>> binomial(n, k).is_hypergeometric(n)
True
```